docs/docs/index.md

```diff
@@ -0,0 +1,25 @@
+<p align="center">
```

Comment:
Why do you place it to `docs/docs/...`?

Reply:
Mkdocs looks for the source documents in a sub-directory names `docs` contained in the directory where the `mkdocs.yml` config file is located. If I move `mkdocs.yml` up to the root of the project then it would only be `docs/...`.

Reply:
Yeah, let's do it

Reply:
Updated :heavy_check_mark: